test/conformance/parallel/consumer-sidecar.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cloud-proxy-sidecar
+  namespace: openshift-ptp
```

Comment:
could you deploy this to test namespace? Its easy to clean on failures

Reply:
fixed